### PR TITLE
Customizable database path

### PR DESCRIPTION
**Pull request by Tolsto**

### CODE REVIEW

File: .tool-versions [hunk 1]
```diff
@@ -1,2 +1,2 @@
-elixir 1.11.0-otp-23
-erlang 23.1.1
+elixir 1.12.1-otp-24
```
Comment (cristianriano) [inline .tool-versions]: Don't we have Elixir 1.11 in production? I think is better to use the prd version

**Review by cristianriano — APPROVED** [review]

File: .tool-versions (hunk 1 above)
Comment (Tolsto) [reply .tool-versions]: PEA will use 1.12 in production once we migrate to Kubernetes.

Comment (danirukun) [reply .tool-versions]: Is it ready to merge then? @Tolsto

**Review by peterkrenn — APPROVED** [review]

**Review by peterkrenn — CHANGES_REQUESTED** [review]
We need to also update the version in `mix.exs` and tag it.

**Review by peterkrenn — APPROVED** [review]

**Pull request merged**